### PR TITLE
completed hw1 bubble sort

### PR DESCRIPTION
**Pull request by romakonn1**

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dinamio) [inline .idea/misc.xml]: Files under .idea should not be in the pull request

**Review by dinamio — CHANGES_REQUESTED** [review]

File: .idea/misc.xml (hunk 1 above)
Comment (romakonn1) [reply .idea/misc.xml]: i will use git ignore, correct?

File: SortLab/SortLab.iml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
```
Comment (dinamio) [inline SortLab/SortLab.iml]: This file doesn't need to be here

**Review by dinamio — CHANGES_REQUESTED** [review]

**Review by dinamio — CHANGES_REQUESTED** [review]
Why you deleted all from others readme?

**Pull request closed without merging**